Library/Homebrew/cmd/cleanup.rb

```diff
@@ -22,7 +22,8 @@ def cleanup_args
         #{days} days old. This can be adjusted with `HOMEBREW_CLEANUP_MAX_AGE_DAYS`.
       EOS
       flag   "--prune=",
-             description: "Remove all cache files older than specified <days>."
+             description: "Remove all cache files older than specified <days>."\
+                          "if you want to remove everything, use `--prune=all`."
```

Comment:
```suggestion
             description: "Remove all cache files older than specified <days>. "\
                          "If you want to remove everything, use `--prune=all`."
```

Reply:
Thank you for the review. I fix it right now.